spaceship.zsh

```diff
@@ -76,13 +76,27 @@ SPACESHIP_DIR_COLOR="${SPACESHIP_DIR_COLOR:="cyan"}"
 SPACESHIP_GIT_SHOW="${SPACESHIP_GIT_SHOW:=true}"
 SPACESHIP_GIT_PREFIX="${SPACESHIP_GIT_PREFIX:="on "}"
 SPACESHIP_GIT_SUFFIX="${SPACESHIP_GIT_SUFFIX:=""}"
-SPACESHIP_GIT_UNCOMMITTED="${SPACESHIP_GIT_UNCOMMITTED:="+"}"
-SPACESHIP_GIT_UNSTAGED="${SPACESHIP_GIT_UNSTAGED:="!"}"
-SPACESHIP_GIT_UNTRACKED="${SPACESHIP_GIT_UNTRACKED:="?"}"
-SPACESHIP_GIT_STASHED="${SPACESHIP_GIT_STASHED:="$"}"
-SPACESHIP_GIT_UNPULLED="${SPACESHIP_GIT_UNPULLED:="⇣"}"
-SPACESHIP_GIT_UNPUSHED="${SPACESHIP_GIT_UNPUSHED:="⇡"}"
-SPACESHIP_GIT_COLOR="${SPACESHIP_GIT_COLOR:="magenta"}"
+SPACESHIP_GIT_SYMBOL="${SPACESHIP_GIT_SYMBOL:=" "}"
```

Comment:
It looks like this:

![image](https://cloud.githubusercontent.com/assets/3459374/25776213/2e2b9b1a-32c1-11e7-9c5d-3c21c54001c6.png)

I've added this icon because it looks nice and it would be more obvious along with `hg`'s icon.